ktor-io/jvm/test/io/ktor/utils/io/BlockingAdaptersOnProhibitedThreadTest.kt

```diff
@@ -13,6 +13,7 @@ class BlockingAdaptersOnProhibitedThreadTest {
 
     @AfterTest
     fun cleanup() {
+        fail()
```

Comment:
Can you explain why you fail tests here?

Reply:
Thanks for catch, was added by mistake